### DISCUSSION TITLE
can it run as a desktop app on solara

### DISCUSSION THREAD

**Discussion by bobwatcherx** [widgetti/solara#100] [Q&A]
is there a way to run as desktop mode on solara and not run for web

**Comment by maartenbreddels** [comment]
Can you be more specific, or do you have examples to make us better understand what you want?

**Reply by bobwatcherx** [reply]
when I run solara run sol.py does it work in desktop mode. so a window will appear like a desktop application. and can Solara be built onto the desktop

**Reply by maartenbreddels** [reply]
Opened an issue at https://github.com/widgetti/solara/issues/114
